MapleServer2/Data/Static/MapMetadataStorage.cs

```diff
@@ -146,8 +147,21 @@ public static int GetDistanceToNextBlockBelow(int mapId, CoordS coord, out MapBl
         return 99;
     }
 
-    public static bool MapIsInstancedOnly(int mapId)
+    public static bool IsInstancedOnly(int mapId)
     {
+        if (mapId is >= 50000001 and <= 50000014) // lobbyIds are capacity > 0, yet are instanced only
+        {
+            return true;
+        }
+
+        switch (mapId)
+        {
+            case (int) Map.RosettaBeautySalon:
+            case (int) Map.TriaPlasticSurgery:
+            case (int) Map.DouglasDyeWorkshop:
+                return true;
+        }
```

Comment:
shouldn't already be handled with metadata stuff?

Reply:
this was needed so that the function could be consistently used in other places were at logout and other places players are put into a safe map when logging out and so on, they would not at these places before the change